datawell/iex.py

```diff
@@ -40,6 +40,33 @@ def get_stocks(self):
             ex = app.AppException(e, message)
             raise ex
 
+    def get_financials(self, ticker: str = ""):
+        """
+        Will return financials data (either all, or for given ticker).
+        :type ticker: company id to load financial data
+        :return: list financial data as list(), raises AppException if encountered an error
+        """
+
+        fin_list = []
+        for entry in self.Symbols:
+            try:
+                uri = f'{app.BASE_API_URL}stock/{entry["symbol"]}/financials/{app.API_TOKEN}'
+                if entry["symbol"] == ticker.upper():
+                    return [self.load_from_iex(uri)["financials"]]
+                else:
+                    fin_list.append(self.load_from_iex(uri)["financials"])
+
+            except KeyError:
```

Comment:
Very pythonic exception handling!
Unlike Java or C#, which by default view an exception as something unusual and thus to be avoided, python intends exceptions as a means of communication. it will good is you'll have a short discussion with colleagues pointing that out - exceptions are not to be feared - build your own and communicate/use them.